PatientRegistrationApp/src/customerSettings.js

```diff
@@ -1,16 +1,16 @@
 export default {
     householdDefinition: 'A household is composed of any people who occupy the same given housing unit. If you are planning to register your household, all members must currently be residing in the same address. If members of a household are residing in other housing units, it is necessary to register any given members individually. Max household registration of 20 registrants.',
-    
+
     contactInfoDisclosure: 'We ask that you please provide your phone number and/or e-mail address.  This contact information will provide us with the ability to send you important information regarding your vaccination registration, follow-up appointment, and vaccination summary.',
     //contactInfoDisclosure: 'We ask that you please provide your phone number and/or e-mail address. This will make it possible for us to send you information about your vaccination registration, follow-up appointment, and vaccination summary.',
-    
+
     contactInfoConsequence: 'By withholding your e-mail and/or phone number, it will not be possible to provide you with the important vaccination information.',
     //contactInfoConsequence: 'This information will allow us to follow up on any details that need to be handled quickly.',
-    
+
     contactInfoAcknowledgement: 'You understand that we will use the phone number and e-mail that you provide to contact you with vaccination registration and follow-up information.',
 
     welcome: 'Welcome!',
-    introStatement: 'You are about to register as a patient for the COVID-19 vaccine. This form will securely collect your patient information, and you have the option to register your entire household to save time. You will also be asked vaccination Phase eligibility questions. At the time of vaccination, you may be required to bring proof of employment at a medical facility, etc.',
+    introStatement: 'You are about to register as a patient for the COVID-19 vaccine. This form will securely collect your patient information. At the time of vaccination, you may be required to bring proof of employment at a medical facility, etc.',
     expectationsStatement: 'After registering as a patient, you will receive a notification to schedule your COVID vaccination appointment if you are not already eligible to do so based upon the CDC and local Phases. Upon receiving your eligibility notification, you will answer COVID screening questions, provide consent, and choose your date to receive the COVID-19 vaccine. You will also be provided with a QR code that contains a patient identifier so that when you arrive at the vaccine clinic, staff can easily look up your patient information.',
```

Comment:
Honestly while I'm in here, might suggest uncapitalizing Phases (cool with being "out of scope" though).